### ISSUE TITLE
EL/M-2106 Radar

### ISSUE THREAD

**Issue by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#3329]
### DB Selector

DB3K

### Baseline

DB3k Sensor 2978

### Hypothetical

No

### Name

EL/M-2106

### Sensor Generation

Late 1970s

### Range (nm)

8.6

### Horizontal Arc (deg)

360

### Sensor Capabilities

Air Search, Range Information

### Search/Track Frequencies

D Band (1-2 GHz) [Old L Band] 15-30 cm

### Target Altitude (ft)

8000 AGL

### Peak Power (W)

150

### Pulse Width (μs)

6.6

### PRF (Hz)

153,000,000

### Comments

Well, I did my best. I'm not sure what an MTI Improvement Factor is.

### Sources

![image](https://user-images.githubusercontent.com/109925704/233802756-d2b78e67-787e-4b5e-a9b5-8e8e90b47d41.png)
![image](https://user-images.githubusercontent.com/109925704/233802766-d2764c41-b243-4d53-a67d-3f45639ed788.png)

**Comment by HVAA** [comment]
Already exists DB sensor 3248

**Comment by FrangibleCover** [comment]
> Already exists DB sensor 3248

Doesn't, the EL/M-2106 ATAR is a completely different radar with a similar role. Elta have very unhelpfully used the name for three different radars, and the other two are the ones we have.

Incidentally, what does one have to do to get a milestone on the tracker?

**Comment by HVAA** [comment]
Just pointing it out, I'll leave it for the team to officially decide. The reality is these are all the same radar, with the same basic role. The Janes entry merely confirms it as far as I can see.  ELTA 2106 family of LAADS (Low Altitude Air Defence System) or ATAR or AN/UPS-3 TDAR...  all have the same pedigree and are part of the same family.

**Comment by PygmalionOfCyprus** [comment]
I'm like...85% sure they are, in fact, the same radar. For now I'm going to use the existing entry. Lacking any data on the differences a new entry would just be a copy, anyway.

**Comment by FrangibleCover** [comment]
I am 100% sure these are not the same radar. Elta and IAI use the '2106' designation for any D-band tactical air search radar they produce. There are four radars marketed under the EL/M-2106 banner, which are related and similar in role, but not the same piece of equipment. The ATAR is this system:
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/550664b8-2bfe-4749-8c24-17bbb388758e)
Note the seven closely spaced horizontal waveguides.
https://www.iai.co.il/p/elm-2106-atar
IAI describe it as being a "fourth generation 3D solid state L-band tactical [air defense radar](https://www.iai.co.il/defense/air/air-defense-systems), designed to support air defense, early warning and traffic control missions. Featuring fully digital beam forming in elevation, the ATAR provides accurate target 3D position, differentiates between aircraft and helicopters and classifies the helicopter type."
In game it has an 18nmi rated range (33km, slightly below the 40km IAI rate it at for helicopters and small aircraft). It has the AESA tag in accordance with its digital beam forming, gives accurate 3D position (compare to the 2D display with 1km range/ 10 deg azimuth uncertainty described above) and is generally a very modern radar. I can't find out when these things entered service, but it's worth noting that it's not mentioned at all in the old Janes information I found above. In game it's late 90s technology.

https://media.discordapp.net/attachments/404743982132887553/1099403179436806236/DexEfJB.jpg?width=1528&height=1074
This is the original EL/M-2106, displayed in what is definitely quite an old reproduction of a photo.
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/8428452e-5db4-48da-8a59-c6790d3f9013)
Another view of it on a Ratel in the Angolan bush, claimed by Alta Pete (https://www.gunners.org.za/wp-content/uploads/2021/12/SOUTH-AFRICA-AIR-DEFENCE-ARTILLERY-CORPS-HISTORY.pdf) to be from 1986 and certainly taken with the awful orange-colour film that all South African images from that war seem to have used.

This is, without question, a totally different radar operating on older principles. It is the described one in the Janes article above, being small enough to be man-portable.
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/583af573-4931-4059-b9b4-09202f1997c9)
Here it is on the quadripod mount described above, in South African service again. As stated in #3440, one of these systems was captured at Goose Green in 1982, making it significantly older than the late 90s tech of the ATAR in game. The rated range is also shorter, only 8.6 nmi against a 2sqm target.

For the sake of completeness, here is the early 90s EL/M-2106H (AN/UPS-3 TDAR), with four horizontal wave guides:
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/916c391f-9c8e-4f79-a929-0560efe28239)
These are quite often confused for other M-2106 radars, probably because they're the easiest ones to get pictures of.

Here is the EL/M-2106NG, a different system again using a true flat panel sensor:
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/facd0242-1b2d-45ce-8062-8c8fc910f93f)

In conclusion, please reopen this request. There is plenty of data to differentiate this system from its three younger siblings, all of which have separate entries in game.

**Comment by PygmalionOfCyprus** [comment]
Aha, I see what's going on here.

The entry built in the DB as the "2106 ATAR" erroneously has some characteristics of the older version, e.g. it's marked as "2D", etc. So, looking at it, I assumed it was representing the correct radar, but really it's not representing anything properly. 

100% handily beats 85%. I'm going to rework the old entries and create a new one for the 2106 as you requested.
